pipelines/deployer/deployer.yaml

```diff
@@ -626,6 +626,7 @@ jobs:
     params:
       env_name: ((account-name))
       terraform_source: platform/pipelines/deployer
+      plan_only: true
```

Comment:
could we add plan_only to the other puts too until we are happy it's safe to remove ie `put: user-state`

And also since plan_only will presumably PASS not FAIL ... we _may_ want to add an explicit task that fails at the end of the deploy job to stop it continuing .... but I don't feel strongly about that... it should be perfectly safe for it to continue